requirements_test.txt

```diff
@@ -1,47 +1,43 @@
-# linters such as flake8 and pylint should be pinned, as new releases
-# make new things fail. Manually update these pins when pulling in a
-# new version
-
-# types-* that have versions roughly corresponding to the packages they
-# contain hints for available should be kept in sync with them
-
+# Automatically generated by gen_requirements_all.py, do not edit
```

Comment:
This is considered the source file for test dependencies.

Reply:
Not anymore, as of this PR. See my reply on the `pyproject.toml` comment for motivation.

Reply:
Unresolved conversation as it is not resolved.